docs/includes/v3_backup_steps_common.rst

```diff
@@ -0,0 +1,146 @@
+
+#. Move the existing *Admin Workstation* SecureDrop code out of the way, by
+   opening a Terminal via **Applications > System Tools > Terminal** and
+   running the command:
+
+   .. code:: sh
+
+      mv ~/Persistent/securedrop ~/Persistent/sd.bak
+
+#. Move the existing *Admin Workstation* SSH configuration out of the way via
+   the Terminal, using the commands:
+
+   .. code:: sh
+
+       mv ~/.ssh/config ~/.ssh/config.bak
+       mv ~/.ssh/known_hosts ~/.ssh/known_hosts.bak
+
+#. Reinstall SecureDrop  on the *Admin Workstation* using the following Terminal
+   commands:
+
+   .. code:: sh
+
+      cd ~/Persistent
+      git clone https://github.com/freedomofpress/securedrop
+
+#. Verify that the current release tag was signed with the release signing key:
+
+   .. code:: sh
+
+      cd ~/Persistent/securedrop/
+      git fetch --tags
+      git tag -v 1.7.1
+
+   The output should include the following two lines:
+
+   .. code:: sh
+
+      gpg:                using RSA key 22245C81E3BAEB4138B36061310F561200F4AD77
+      gpg: Good signature from "SecureDrop Release Signing Key"
+
+
+   .. important::
+      If you do not see the message above, signature verification has failed
+      and you should **not** proceed with the installation. If this happens,
+      please contact us at securedrop@freedom.press.
+
+
+   Verify that each character of the fingerprint matches what is on the
+   screen of your workstation. If it does, you can check out the new release:
+
+   .. code:: sh
+
+      git checkout 1.7.1
+
+   .. important::
+      If you see the warning ``refname '1.7.1' is ambiguous`` in the
+      output, we recommend that you contact us immediately at
+      securedrop@freedom.press (`GPG encrypted <https://securedrop.org/sites/default/files/fpf-email.asc>`__).
+
+#. Copy the old instance's configuration files and backup from ``~/Persistent/sd.bak`` into ``~/Persistent/securedrop`` using the following Terminal commands:
+
+   .. code:: sh
+
+      cd ~/Persistent/securedrop
+      export SD_OLD="~/Persistent/sd.bak/install_files/ansible-base"
+      export SD_NEW="~/Persistent/securedrop/install_files/ansible-base"
+      cp $SD_OLD/group_vars/all/site-specific $SD_NEW/group_vars/all/site-specific
+      cp $SD_OLD/tor_v3_keys.json $SD_NEW/tor_v3_keys.json
+      cp $SD_OLD/sd-backup-old.tar.gz $SD_NEW/sd-backup-old.tar.gz
+
+   You will also need to copy the old instance's *Submission Public Key*,
+   *Ossec Alert Public Key*, and, if configured, the *Journalist Alert Public Key*.
+   Assuming that these are named ``SecureDrop.asc``, ``ossec.asc``, and
+   ``journalist.asc`` respectively, run the following commands:
+
+   .. code:: sh
+
+      cp $SD_OLD/SecureDrop.asc $SD_NEW/SecureDrop.asc
+      cp $SD_OLD/ossec.asc $SD_NEW/ansible-base/ossec.asc
```

Comment:
Since `$SD_NEW` includes `ansible-base` I believe it is duplicated here.

Reply:
You could use the shorter `$SD_NEW/` for all commands that use the same filename as the original.

Reply:
fixed in 2e6a9b3